src/stravalib/tests/unit/test_model.py

```diff
@@ -225,6 +225,23 @@ def test_backward_compatible_attribute_lookup(
         assert not hasattr(lookup_expression, "bound_client")
 
 
+@pytest.mark.parametrize(
```

Comment:
gosh i wish i knew about pytest.mark.parametrize a long time ago 🙃  this is so cool that you can just pass it many options without having to write separate methods (or without creating some sort of loop).